demos/thirdparty/gflags/test/gflags_build.py.in

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
This is 3rdparty code, please leave it as it is.

Reply:
I  have send this change to repo gflags ,  but it have not been updated since 2018.  so I think maybe it is hard to be merge.   And I will  remove change of this file from this PR